src/control/cmd/dmg/pretty/printers.go

```diff
@@ -155,3 +155,41 @@ func PrintResponseErrors(resp hostErrorsGetter, out io.Writer, opts ...PrintConf
 
 	return nil
 }
+
+// PrintErrorsSummary generates a human-readable representation of the supplied
+// HostErrorsMap summary struct and writes it to the supplied io.Writer.
+func UpdateErrorSummary(resp hostErrorsGetter, cmd string, out io.Writer, opts ...PrintConfigOption) error {
+	if resp == nil {
```

Comment:
I am probably missing something with golang, but I do not understand how it could be equal to nil as resp is not a Pointer.

Reply:
resp is *control.CollectLogResp. so it will work.

Reply:
hostErrorsGetter sounds like an interface name. The underlying type behind the interface is always a pointer. :)

Reply:
Thanks for the information.
I did not know that type behind interface is always a pointer: good to know :)